### PR TITLE
Report the js limits in jsz output

### PR DESCRIPTION
**Pull request by ripienaar**
Reports some often requested information about JS config in jsz

Signed-off-by: R.I.Pienaar <rip@devco.net>

### CODE REVIEW

File: server/opts.go [hunk 1]
```diff
@@ -233,10 +233,10 @@ type RemoteLeafOpts struct {
...
 
 type JSLimitOpts struct {
-	MaxRequestBatch int
-	MaxAckPending   int
-	MaxHAAssets     int
-	Duplicates      time.Duration
+	MaxRequestBatch int           `json:"max_request_batch,omitempty"`
+	MaxAckPending   int           `json:"max_ack_pending,omitempty"`
+	MaxHAAssets     int           `json:"max_ha_assets,omitempty"`
+	Duplicates      time.Duration `json:"duplicates,omitempty"`
```
Comment (ripienaar) [inline server/opts.go:239]: I didnt do it here but propose we rename this to something more descriptive MaxDuplicateWindow maybe

Comment (derekcollison) [reply server/opts.go:239]: You can change the json filed name and have clients match that..

**Review by derekcollison — APPROVED** [review]
LGTM - let me know if you want changes before merge.

**Comment by ripienaar** [comment]
Ok I will update that json key in a few hours

**Comment by ripienaar** [comment]
I changed it, so good to go I think @derekcollison

**Pull request merged**